### PR TITLE
Fix crash caused when Discord channel cannot be fetched by channelID 

### PR DESCRIPTION
**Pull request by fantinodavide**
Handle discord channel not found exception in DiscordBasePlugin.prepareToMount() and DiscordBasePlugin.sendDiscordMessage()

### CODE REVIEW

**Review by ect0s — APPROVED** [review]
Looks good to me

The one edge case that may not be covered, is a lack of output as to why the channel couldn't be found, this might make it harder to diagnose issues in regards to Bot Channel Permissions. 

Perhaps we should log the Error stack as well?

**Comment by werewolfboy13** [comment]
Ignore that trying to approve on mobile.

**Comment by werewolfboy13** [comment]
This will need a better commit name before I enable auto merge. If no one is able to do that I'll attempt to take care of it tomorrow. Minus any other changes requested.

**Comment by fantinodavide** [comment]
> Looks good to me
> 
> The one edge case that may not be covered, is a lack of output as to why the channel couldn't be found, this might make it harder to diagnose issues in regards to Bot Channel Permissions.
> 
> Perhaps we should log the Error stack as well?

Let me know if the new [commit](https://github.com/Team-Silver-Sphere/SquadJS/pull/266/commits/e2a9c4b79f98770aa1f2cb2cd8e7d6d460ea77e8) may be a good solution.

**Comment by fantinodavide** [comment]
> This will need a better commit name before I enable auto merge. If no one is able to do that I'll attempt to take care of it tomorrow. Minus any other changes requested.

PR renamed and requested changes (hopefully) satisfied.

**Review by ect0s — DISMISSED** [review]

File: squad-server/layers/layer.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class Layer {
           respawnDelay: vehicle.respawnTime
         })),
         numberOfTanks: (data[t].vehicles || []).filter((v) => {
-          return v.icon.match(/tank/);
+          return v.icon.match(/_tank/);
```
Comment (werewolfboy13) [inline squad-server/layers/layer.js]: Why is this in this PR? Please make seperate PRs for each issue.

**Review by werewolfboy13 — CHANGES_REQUESTED** [review]

**Comment by werewolfboy13** [comment]
If there is any updates to the layers.js needs to be approved by me since I am maintaining that pipeline. Also your force pull just broke your changes.

**Comment by fantinodavide** [comment]
Apologizes for what I did, I tried to fix it as soon as I noticed that I was breaking everything.
New changes to Layers.js have been moved to a new PR

**Review by werewolfboy13 — APPROVED** [review]

File: squad-server/layers/layer.js (hunk 1 above)
Comment (fantinodavide) [reply squad-server/layers/layer.js]: I pushed the commits on the wrong branch, now it should be fixed.

**Review by ect0s — APPROVED** [review]

**Pull request merged**